test/blackbox-tests/test-cases/subst/broken-symlink.t

```diff
@@ -0,0 +1,19 @@
+dune subst should not fail when encountering broken symlinks.
+See #9593.
+
+  $ git init -q
+  $ cat > dune-project << EOF
+  > (lang dune 1.0)
+  > (name proj)
+  > (package
+  >  (name proj))
+  > EOF
+  $ ln -s nonexistent broken 
+  $ git add dune-project broken
```

Comment:
This git repo is needed to get a version for `dune subst` right? Would be good to clarify in the test.